.travis.yml

```diff
@@ -8,18 +8,13 @@ matrix:
   include:
   - os: linux
     dist: trusty # Ubuntu 14.04
-    dotnet: 1.0.0-preview2-1-003177
+    dotnet: 1.0.1
     mono: none
     env: DOTNETCORE=1
     sudo: required
-  - os: osx
-    osx_image: xcode7.3 # macOS 10.11
-    dotnet: 1.0.0-preview2-1-003177
-    mono: none
-    env: DOTNETCORE=1
 
 install:
-   - sed -i.bak 's/^.*\"net451\":.*$//g' src/Confluent.Kafka/project.json
+   - sed -i.bak 's/net451;//g' src/Confluent.Kafka/Confluent.Kafka.csproj
```

Comment:
nit: Perhaps provide more context to this replacement to avoid future inadvertent matches?